tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.h

```diff
@@ -106,6 +110,10 @@ class GpuCudaMallocAsyncAllocator : public Allocator {
   CUmemoryPool pool_;
 #endif  // TF_CUDA_MALLOC_ASYNC_SUPPORTED
 
+  // Just a counter for the number of time this class is instanciated.
+  // Only useful for tests.
+  static int nb_instanciated_;
```

Comment:
Does this need to be an `std::atomic`?

Reply:
Implemented. Note the current TF GpuDevice create them serially. But having it atomic is more future proof.